### PR TITLE
Coaching advice

### PR DESCRIPTION
**Pull request by spacekookie**
I still wanna change the structure of the points and maybe talk a bit more about peer-learning

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
one favor: could u move this into resources please? thanks!

File: resources/coaching-advice.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- Be positive and encouraging
+  - It's okay to make mistakes
+  - And take breaks if they get frustrated!
+  - Leave time for experimentation and "playing around"
+  - If people are getting "off the path", don't force them back on!
+- Make sure people are having fun
+
+### General teaching advice
+
+Others are generally good advice when teaching anything (even in a more hands-off approach)
```
Comment (ghost) [inline resources/coaching-advice.md]: "others"?

File: resources/coaching-advice.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- Create a friendly and welcoming environment
+  - Use students names and pronouns
+  - Smile at them, make eye contact
+- Admit if you don't know something; that's okay!
+- When explaining, try to avoid jargon, especially on tech subjects
+  - Don't assume pre-existing knowledge
+  - Try to rephrase things in case of misunderstandings
+- Don't accept students self-deprecation
```
Comment (ghost) [inline resources/coaching-advice.md:30]: `students'`

File: resources/coaching-advice.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+- This isn't about you, it's about the students; go at their pace
+- Talk slowly, leave space after saying something
+- When answering questions be sure to leave them time to think about it. Then make sure they understood it
+
+
+### **Don'ts**
+
+- Follow the code of conduct of your event, **be an example that students can follow!**
+  - This means no harassment, no sexual advances, no slurs or violating participants boundries
```
Comment (ghost) [inline resources/coaching-advice.md:59]: additionally, if you see any of your fellow teachers/mentors/organizers doing this, confront them immediately

File: resources/coaching-advice.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- Don't roll your eyes at questions, don't be annoyed by misunderstanding, don't be impatient
+- Don't start debates about programming languages, toolkits, etc.
+  - Avoid being opinionated
+- **Don't touch their keyboards!**
```
Comment (ghost) [inline resources/coaching-advice.md]: or screens!

**Review by ghost — APPROVED** [review]
this is super cool!

File: resources/coaching-advice.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
+# Coaching Advice
+
+The following material isn't meant as rules but as guidelines. Feel free to pick and chose advice from different points as you see fit for the mentors and student audience alike. The material also aims for an explorative learning approach (sometimes called *the hard way*). This puts emphasis on letting students discover things by themselves, then helping them through what it means.
```
Comment (yawboakye) [inline resources/coaching-advice.md]: s/chose/choose

File: resources/coaching-advice.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+---
+
+There is a fundamental difference between a **coach** and a **teacher**. The goal isn't to stand in front of a class and lecture them about Rust but to stand *besides* them and helping them discover the language by themselves. Some things to consider are specific to teaching programming (or other tech related fields) and having experience with that is always a plus.
```
Comment (yawboakye) [inline resources/coaching-advice.md]: "and help them discover"?

**Review by yawboakye — COMMENTED** [review]
@spacekookie See if the suggestions make sense to you

**Pull request merged**